studio/components/layouts/ProjectLayout/LayoutHeader/NotificationsPopover/NotificationRow.tsx

```diff
@@ -28,10 +29,11 @@ const NotificationRow: FC<Props> = ({
   onSelectRollbackMigration,
   onSelectFinalizeMigration,
 }) => {
-  const { app, ui } = useStore()
+  const { ui } = useStore()
```

Comment:
GH doesnt let me comment on lines that didn't change 😛, but we can change Props to NotificationRowProps and remove FC